app/assets/javascripts/dialog-holder/addon/services/dialog.js

```diff
@@ -106,30 +106,38 @@ export default Service.extend({
       ...params,
       shouldDisplayCancel: true,
       buttons: null,
-      type: "confirm",
+      type: "confirm"
     });
   },
 
-  notice(message) {
+  notice(params) {
+    // support string param for easier porting of bootbox.alert
```

Comment:
Another option to modifying the `notice` is to revert to using `this.dialog.alert` here and pass a custom `buttons` array  to it. The dialog service already supports that.

Reply:
I've been looking into this and the issue I have been running into mentally is that pulling the button out of the settings-editor.js component class seems tricky. 

Here's how I'm thinking about it, and this may be wrong: It would mean that the @save handler would have to be in the admin-customize-themes-show controller ... which would mean having communicating between the component and the controller in a number of ways, including updating save state for hiding/unhiding the button, a way of extracting the contents of the ace editor into the controller, communicating validations back into the popup etc. perhaps this is more straightforward than I am thinking?